pytorch_binding/test/new_benchmark.py

```diff
@@ -0,0 +1,158 @@
+import sys
+import argparse
+import pickle
+
+import torch
+import torch.nn as nn
+from torch.profiler import profile, record_function, ProfilerActivity
+
+T = torch.Tensor
+
```

Comment:
Please don't do that. It's confused with the time dimension.

Reply:
fixed.